mkosi/util.py

```diff
@@ -257,3 +258,42 @@ def current_home_dir() -> Optional[Path]:
 
 def unique(seq: Sequence[T]) -> list[T]:
     return list(dict.fromkeys(seq))
+
+
+def mandatory_variable(name: str) -> str:
+    try:
+        return os.environ[name]
+    except KeyError:
+        die(f"${name} must be set in the environment")
+
+
+def kernel_install_arg_parse(parser: argparse.ArgumentParser) -> None:
```

Comment:
Same thing here, just keep it in initrd.py, makes more sense there for sharing than here